### PR TITLE
Chore(dep): update the tfsec version

### PR DESCRIPTION
**Pull request by owenrumney**
- Latest tfsec version has many more checks and also includes the AVD ID

### CODE REVIEW

**Review by liamg — APPROVED** [review]

File: config/scanner/scanner_test.go [hunk 1]
```diff
@@ -72,40 +72,40 @@ func TestScanner_ScanConfig(t *testing.T) {
...
 							Severity:           "CRITICAL",
+							RecommendedActions: "Don't include sensitive data in blocks",
 						},
 					},
 					{
 						Message: `Resource 'azurerm_managed_disk.source' passed check: Potentially sensitive data stored in block attribute.`,
 						PolicyMetadata: types.PolicyMetadata{
 							Type:               "Terraform Security Check powered by tfsec",
-							ID:                 "GEN003",
+							ID:                 "AVD-GEN-0001",
```
Comment (knqyf263) [inline config/scanner/scanner_test.go:108]: I couldn't find this ID in AVD. Is it planned to be added?
https://avd.aquasec.com/

Comment (owenrumney) [reply config/scanner/scanner_test.go:108]: It's here - https://github.com/aquasecurity/cloud-metadata/blob/main/general/secrets/sensitive-in-attribute/metadata.json 

The idea of this repo is that all IAC and CSPM rules will share the same set of common metadata but have different remediation steps depending on the reason (cli to resolve issues already in production maybe, or the required changes to terraform or cloudformation to fix the issue in code)

Comment (knqyf263) [reply config/scanner/scanner_test.go:108]: Do you mean AVD will not have the ID? Should Trivy link these IDs to TFsec doc directly? I thought all information would be aggregated in AVD and Trivy should link IDs to AVD.

Comment (knqyf263) [reply config/scanner/scanner_test.go:108]: OK, thanks!

Comment (knqyf263) [reply config/scanner/scanner_test.go:108]: Trivy currently refers to the tfsec documentation (tfsec.dev). In other words, this change would not be a regression in terms of references. I think we can merge this PR now and will switch references to AVD once AVD will be ready.

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
Seems like quay.io is down now. I will retry it later.

**Comment by knqyf263** [comment]
Thanks!

**Pull request merged**